_posts/2018/2018-09-16-pluck-can-save-a-bunch-of-memory.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Using pluck can save a bunch of memory
+description: ''
+layout: article
+category: ruby
+image:
+  base: '2018/using-pluck-can-save-a-bunch-of-memory'
+  alt: 'Guitarist'
+  source: 'https://unsplash.com/photos/DUrdHDn81mk'
+  credit: 'Jacek Dylag'
+---
+
+Active Record models are incredible flexible and provide a very large amount of flexibility. One of the consequences of providing this large API of methods is that each individual Active Record object takes up a lot of space when loaded into memory.
+
+If you only use a single field, or just need the `id`s, Rails has your back.
```

Comment:
Why not? "If you only use a single field, for example, the `id`s, then Rails has your back."?